embedded/common/modules/sensor-drivers/Driver_Common.h

```diff
@@ -0,0 +1,72 @@
+/* -----------------------------------------------------------------------------
```

Comment:
Sure. Moving it to `external\CMSIS\Driver\Include`. The location will be same in GPIO commit too.